README.md

```diff
@@ -685,53 +669,40 @@ nock('http://my.server.com')
   .reply(200, '<html></html>')
 ```
 
-`delay()` could also be used as
+`delay(1000)` is an alias for `delayConnection(1000).delayBody(0)`  
+`delay({ head: 1000, body: 2000 })` is an alias for `delayConnection(1000).delayBody(2000)`  
+Both of which are covered in detail below.
 
-```
-delay({
-   head: headDelayInMs,
-   body: bodyDelayInMs
-})
-```
+#### Delay the connection
 
-for example
+You are able to specify the number of milliseconds that your connection should be idle before it starts to receive the response.
+To simulate a socket timeout, provide a larger value than the timeout setting on the request.
 
 ```js
 nock('http://my.server.com')
   .get('/')
-  .delay({
-    head: 2000, // header will be delayed for 2 seconds, i.e. the whole response will be delayed for 2 seconds.
-    body: 3000, // body will be delayed for another 3 seconds after header is sent out.
-  })
+  .delayConnection(2000) // 2 seconds
   .reply(200, '<html></html>')
-```
 
-### Delay the connection
+req = http.request('http://my.server.com', { timeout: 1000 })
+```
 
-`delayConnection(1000)` is equivalent to `delay({ head: 1000 })`.
+The [`'timeout'`](https://nodejs.org/api/http.html#http_event_timeout) will be emitted almost immediately, so your tests don't have to wait for long timeout tests.  
+However, if you're not testing the timeout event, this setting will still wait the desired time before emitting the [`'response'`](http://nodejs.org/api/http.html#http_event_response).
 
-### Socket timeout
+#### Delay the response body
 
-You are able to specify the number of milliseconds that your connection should be idle, to simulate a socket timeout.
+You are able to specify the number of milliseconds that the response body should be delayed.  
+This is the time between the headers being received and the body starting to be received.
 
 ```js
 nock('http://my.server.com')
   .get('/')
-  .socketDelay(2000) // 2 seconds
+  .delayBody(2000) // 2 seconds
   .reply(200, '<html></html>')
 ```
 
-To test a request like the following:
-
-```js
-req = http.request('http://my.server.com', res => {
-  ...
-})
-req.setTimeout(1000, () => { req.abort() })
-req.end()
-```
-
-NOTE: the timeout will be fired immediately, and will not leave the simulated connection idle for the specified period of time.
+The timer starts after the [`'response'`](http://nodejs.org/api/http.html#http_event_response) event and will delay the [IncomingMessage](http://nodejs.org/api/http.html#http_http_incomingmessage) from emitting its first `'data'` or `'end'` event.
```

Comment:
Does this one cause a real delay?

Reply:
It does.

Reply:
Hmm, I think it is confusing for the caller that one of these methods causes a real delay and the other one doesn't.

Maybe that was true before? Either way, since the semantics of these APIs are changing, I think it is probably a good time to look at it.

One possibility would be to change the names, e.g.:

- `simulateDelayedConnection(ms)`
- `delayBody(ms)`